docs/optimization-framework/ab-testing.md

```diff
@@ -35,20 +31,24 @@ It is enabled by default but can be disabled via the configuration file:
 }
 ```
 
-Within the post edit screen click on the A/B icon to access the Experiments panel. Here you can set the alternative titles, the amount of traffic to show the tests to as well as start and end dates.
+Within the post edit screen click on the A/B icon to access the Experiments panel. Here you can set the alternative titles and/or alternative featured images, the amount of traffic to show the tests to as well as start and end dates.
```

Comment:
Think we go a bit more generic perhaps to avoid having to edit lots of it in future.
```suggestion
Within the post edit screen click on the A/B icon to access the Experiments panel. Here you can set up the variants for your tests, the amount of traffic to show the tests to as well as start and end dates.
```